### ISSUE TITLE
plugin installation issue Entitlements

### ISSUE THREAD

**Issue by krish-dev** [EddyVerbruggen/cordova-plugin-googleplus#288]
While I'm going to add this plugin, getting this error

I have fired this command 

`cordova plugin add cordova-plugin-googleplus --save --variable REVERSED_CLIENT_ID=com.googleusercontent.apps.xxxxxxxx52-xxxxxxxxxxxxxxxxxxx08ss`

Cordova version : 6.3.1
Node version: 4.5.0
npm version: 2.15.9
Xcode version: 8.0 (8A218a)

```
Installing "cordova-plugin-googleplus" for ios
Google Sign-In prerequisites
START Running hook to add iOS Keychain Sharing entitlements (required since iOS 10)
Will add iOS Keychain Sharing entitlements to project 'v4i'
Error: ENOENT: no such file or directory, open '/Users/apple/Documents/htdocs/v4i/platforms/ios/v4i/Resources/v4i.entitlements'
```

**Comment by EddyVerbruggen** [comment]
Hmm, tested this yesterday on Cordova 3.x, perhaps the new hook doesn't work so well with newer Cordova versions. Will tweak it today!

**Comment by EddyVerbruggen** [comment]
Plz check the updated plugin from this repo.

**Comment by EddyVerbruggen** [comment]
Fixed in 5.1.1, now on npm
